src/ert/storage/local_storage.py

```diff
@@ -194,21 +195,26 @@ def __init__(
         with contextlib.suppress(FileExistsError):
             (self.path / ".fs_version").symlink_to("index.json")
 
-        self._lock = FileLock(self.path / "storage.lock")
-        try:
-            self._lock.acquire(timeout=self.LOCK_TIMEOUT)
-        except Timeout as e:
-            raise TimeoutError(
-                f"Not able to acquire lock for: {self.path}."
-                " You may already be running ERT,"
-                " or another user is using the same ENSPATH."
-            ) from e
+        self._ignore_mutex = ignore_mutex
+        if not ignore_mutex:
+            self._lock = FileLock(self.path / "storage.lock")
+            try:
+                self._lock.acquire(timeout=self.LOCK_TIMEOUT)
+            except Timeout as e:
+                raise TimeoutError(
+                    f"Not able to acquire lock for: {self.path}."
+                    " You may already be running ERT,"
+                    " or another user is using the same ENSPATH."
+                ) from e
 
         super().__init__(path)
 
         self._save_index()
 
     def close(self) -> None:
+        if self._ignore_mutex:
```

Comment:
This function is doing "nothing" ??

Reply:
If `self._ignore_mutex` is set, yes. Otherwise it'll save data to disk.

Reply:
This change has been since removed.